go.mod

```diff
@@ -22,9 +22,8 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/stretchr/testify v1.7.0
 	go.opentelemetry.io/collector v0.27.0
-  github.com/opencontainers/runc-1.0.0-rc95
```

Comment:
I remember we were. using it for run as user? https://github.com/aws-observability/aws-otel-collector/blob/main/pkg/userutils/userutil_linux.go#L27